engines/shared/app/errors/shared/error.rb

```diff
@@ -0,0 +1,27 @@
+module Shared
+  class Error < Shared::BaseSerializer
+    ERRORS = {
+      not_found: NotFoundError,
+      bad_request: BadRequestError
+    }
+
+    schema do
+      type 'error'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.